lambdaclass/curse_of_myrra
# Dagna animations

This PR implements Dagna animations for Dagna.

Please test the PR and leave feedbacks for the animation tempo.

To test:

1. Scherzo its just the same as walking animation but slowed
2. Skill1 Dance Macabre , its a little dance in the place
3. Skill2 Pizzicato , a tittle shotgun in the place
4. Skill3 Band recluit , a tittle jump with a tittle movement ( should i remove the movement )
5. Skill4 Orchesta, no animation

closes #749
